flake.nix

```diff
@@ -13,10 +13,15 @@
     pre-commit-hooks.url = "github:cachix/pre-commit-hooks.nix";
 
     lbf.url = "github:mlabs-haskell/lambda-buffers";
+    flake-lang.url = "github:mlabs-haskell/flake-lang.nix";
     haskell-nix.follows = "lbf/haskell-nix";
     iohk-nix.follows = "lbf/iohk-nix";
     ctl.follows = "lbf/ctl";
     plutarch.follows = "lbf/plutarch";
+    crane.url = "github:ipetkov/crane";
+
+    plutip.url = "github:mlabs-haskell/plutip/1bf0b547cd3689c727586abb8385c008fb2a3d1c";
```

Comment:
I tried using pkgsForCtl, but unfortunately that failed on Mac (even on my local machine).
This is the exact version that CTL uses (unfortunately I couldn't use `follows = ctl/plutip` for some misterious reasons)

Reply:
What's the reason? And also, why do you care about using CTL's versions? You can just point at the latest of both? No need to pin for your Rust parts?

Reply:
simply because the latest commit also failed...
the whole ecosystem is broken

Reply:
follows = ctl/plutip failed on ci with evaluation failure. I don't really understand, because it worked locally.